packages/muriel-base-components/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "muriel-base-components",
+  "version": "1.0.0",
+  "description": "The Muriel Base Components",
+  "main": "dist/cjs/index.js",
```

Comment:
No CJS build yet -- should we remove for now?

Reply:
yeah, i guess. this is mostly exploratory so far.